kart/point_cloud/import_.py

```diff
@@ -116,19 +124,14 @@ def point_cloud_import(ctx, sources):
             "crs84_envelope": crs84_envelope,
         }
 
-    click.echo()
+    click.echo(copc_version_set[0])
```

Comment:
this will echo `1`, accidentally left in?

Reply:
Fixed